PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,34 @@
+<!--- Provide a general summary of your changes in the Title above -->
+<!--- See our Contribution Guidelines here:
+          https://github.com/tripal/tripal/blob/7.x-3.x/CONTRIBUTING.md -->
```

Comment:
Maybe add an empty line here so that `Issue #` doesn't get missed?

Reply:
agreed, added